missing-kit-form/form.tex

```diff
@@ -0,0 +1,41 @@
+\documentclass[a4paper, 12pt]{article}
+\usepackage[utf8]{inputenc}
+
+\usepackage[default,osfigures]{opensans}
+
+\usepackage[margin=0.5in]{geometry}
+\setlength{\parindent}{0cm}
+
+\pagenumbering{gobble}
+
+\begin{document}
+
+\begin{center}
+
+    {\huge{Student Robotics Missing Kit Form}}
+
+    \medskip
+
+\end{center}
+
+Please fill out this form if some of the team's kit is missing when they return it at the end of the event. The required parts can be found on the "Kit Contents List". \\
+
+\bigskip
+
+{\large{Team TLA}: \hrulefill} \\
+
+{\large{Volunteer Name}: \hrulefill} \\
+
+{\large{Missing Parts}:}
```

Comment:
Perhaps also record a reason? (e.g: left at school, went missing at the competition)